core/src/main/java/io/substrait/extended/expression/ProtoExtendedExpressionConverter.java

```diff
@@ -0,0 +1,84 @@
+package io.substrait.extended.expression;
+
+import io.substrait.expression.Expression;
+import io.substrait.expression.proto.ProtoExpressionConverter;
+import io.substrait.extension.*;
```

Comment:
Can we explicitly list imports instead of using `*`?

Reply:
This is the result of applying `./gradlew spotlessApply` let me check if there are changes needed